.github/workflows/dependabot-build.yml

```diff
@@ -32,16 +32,14 @@ jobs:
       # Check out using a PAT so any pushed changes will trigger checkruns
       - uses: actions/checkout@v4
         with:
-          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
This was a leftover that I'd forgotten to remove as part of:
* https://github.com/github/dependabot-action/pull/1456

So removed it here.